tests/fixtures/bad_alignments/intermediates/dnadiff/MGV-GENOME-0264574_vs_MGV-GENOME-0357962.report

```diff
@@ -39,49 +39,49 @@ TandemInsAvg                    0.00                 0.00
 AT                          0(0.00%)             0(0.00%)
+AG                          0(0.00%)             0(0.00%)
 AC                          0(0.00%)             0(0.00%)
+CA                          0(0.00%)             0(0.00%)
+CT                          0(0.00%)             0(0.00%)
+CG                          0(0.00%)             0(0.00%)
 TG                          0(0.00%)             0(0.00%)
 TC                          0(0.00%)             0(0.00%)
 TA                          0(0.00%)             0(0.00%)
-
-TotalGSNPs                         0                    0
-GT                          0(0.00%)             0(0.00%)
-GC                          0(0.00%)             0(0.00%)
 GA                          0(0.00%)             0(0.00%)
-TC                          0(0.00%)             0(0.00%)
-TA                          0(0.00%)             0(0.00%)
-TG                          0(0.00%)             0(0.00%)
-CA                          0(0.00%)             0(0.00%)
-CT                          0(0.00%)             0(0.00%)
-CG                          0(0.00%)             0(0.00%)
-AG                          0(0.00%)             0(0.00%)
-AC                          0(0.00%)             0(0.00%)
-AT                          0(0.00%)             0(0.00%)
+GC                          0(0.00%)             0(0.00%)
+GT                          0(0.00%)             0(0.00%)
 
 TotalIndels                        0                    0
+T.                          0(0.00%)             0(0.00%)
 G.                          0(0.00%)             0(0.00%)
 C.                          0(0.00%)             0(0.00%)
 A.                          0(0.00%)             0(0.00%)
-T.                          0(0.00%)             0(0.00%)
-.A                          0(0.00%)             0(0.00%)
-.C                          0(0.00%)             0(0.00%)
 .T                          0(0.00%)             0(0.00%)
 .G                          0(0.00%)             0(0.00%)
+.A                          0(0.00%)             0(0.00%)
+.C                          0(0.00%)             0(0.00%)
 
 TotalGIndels                       0                    0
-G.                          0(0.00%)             0(0.00%)
-T.                          0(0.00%)             0(0.00%)
-C.                          0(0.00%)             0(0.00%)
 A.                          0(0.00%)             0(0.00%)
+C.                          0(0.00%)             0(0.00%)
+T.                          0(0.00%)             0(0.00%)
+G.                          0(0.00%)             0(0.00%)
+.G                          0(0.00%)             0(0.00%)
 .T                          0(0.00%)             0(0.00%)
-.A                          0(0.00%)             0(0.00%)
 .C                          0(0.00%)             0(0.00%)
-.G                          0(0.00%)             0(0.00%)
+.A                          0(0.00%)             0(0.00%)
```

Comment:
Why does this file (and one other report) change so much? I was seeing this myself when I moved all the files on #194.

Reply:
I believe the files contain the same information overall; only the order in which each SNP is reported differs.

Reply:
I just generated the `.report` files for now, but as you suggested at some point, it might be worth deleting them after we generate the matrices, since we don't use `.report` files for anything else. But I think, it would be best to do it on a diffrent branch.

Reply:
OK, but then in future I don't think we don't really need/want to check this pointless change into the git history.

FYI: To fix this you'd need to rewrite the branch history (not worth the hassle?), or we could apply this PR as a single squashed-commit.

Reply:
Or delete them - even tidier 😉